src/components/standup/FormInputComponent.tsx

```diff
@@ -60,11 +52,10 @@ const FormInputComponent: FC<setIsFormVisibleProps> = ({
         e.preventDefault();
         const newData = {
             type: 'user',
-            completed: sections[0].inputs.join('.'),
-            planned: sections[1].inputs.join('. '),
-            blockers: sections[2].inputs.join('. '),
+            completed: sections[0].inputs.join(' ').trim(),
+            planned: sections[1].inputs.join(' ').trim(),
+            blockers: sections[2].inputs.join(' ').trim(),
```

Comment:
- do we need to join with space and then perform a trim? can we just do `join('')`

Reply:
@skv93-coder can you fix this code, as it's blocking my PR

Reply:
Thanks for review, but doing join(“”) creates 

[this is first, this is second] => this is firstthis is second

But with join(“ “).trim()


[this is first, this is second] => this is first this is second

Reply:
- @skv93-coder how will ["this is first", "this is second"] add a space at the start and end of the sentence?

Reply:
No it does not adds any space.

Trim is to remove space from user input